application/modules/quotes/controllers/Ajax.php

```
@@ -102,6 +141,9 @@ public function save()
             // Generate new quote number if needed
             $quote_number = $this->input->post('quote_number');
 
+            $quote_work_order = $this->input->post('quote_work_order');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$quote_work_order'. (undefined)

(UnusedLocalVariable)